ansible_base/rbac/service_api/views.py

```diff
@@ -127,6 +173,10 @@ def assign(self, request):
     def unassign(self, request):
         return self._unassign(request)
 
+    @action(detail=False, methods=['post'], url_path='object_delete')
+    def object_delete(self, request):
```

Comment:
oooh ok ok, I'm worried that by having a custom action name (totally serious here) that the authentication class for service tokens will cause you to get a 401. Because it hard-codes a list of allowed actions, see "resource_api_actions", and the concern is that `object_delete` isn't in that list.

It wasn't obvious to me it would go this way. Depending on how the URL was included I figured it could be a new viewset that has the `create` method and no other CRUD methods, which should get past the bouncer.